safer_remote_clusters.md

```diff
@@ -0,0 +1,43 @@
+# Safer Remote Clusters (proposal)
+
+### Problem
+
+Users often change their kube context, forget about it, `tilt up`, and end up
+deploying to prod.
+
+### Solutions
+
+The following are not mutually exclusive!
+
+#### Point users at a pure Starlark Solution
+
+a la [what dmiller came up with](https://gist.github.com/jazzdan/d30ef73482920beb3ac9fa2fcb2c3146)
+
+Pros:
+* cheap
+* flexible
+
+Cons:
+* not very discoverable
+* unsafe by default
```

Comment:
i think a requirement is to have a safe default

Reply:
I agree completely! I largely included this approach to exclude it for this reason, but I could have been more explicit.